flink/v1.20/flink/src/test/java/org/apache/iceberg/flink/data/AbstractTestFlinkAvroReaderWriter.java

```diff
@@ -65,12 +63,11 @@ public class TestFlinkAvroReaderWriter extends DataTest {
           Types.NestedField.optional(9, "decimal", Types.DecimalType.of(4, 2)));
 
   @Override
-  protected void writeAndValidate(Schema schema) throws IOException {
-    List<Record> expectedRecords = RandomGenericData.generate(schema, NUM_RECORDS, 1991L);
-    writeAndValidate(schema, expectedRecords, NUM_RECORDS);
-  }
+  protected abstract void writeAndValidate(Schema schema) throws IOException;
```

Comment:
is there a particular reason why this is being made abstract now? It seems both subclasses override and do the same thing as this method was doing previously